### PR TITLE
Update permissions.yml Dr.Fate

### PR DESCRIPTION
**Pull request by flamingwarrior10**
Doctor fate perms for power remake

### CODE REVIEW

**Comment by flamingwarrior10** [comment]
more powers scroll down

**Comment by flamingwarrior10** [comment]
http://comicvine.gamespot.com/doctor-fate-kent-nelson/4005-6599/

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -2777,6 +2777,29 @@ groups:
...
+      - ch.alias.run1
+      - ch.alias.run2
+      - ch.alias.run3
+      - ch.alias.run4
+      - ch.alias.jump0
+      - ch.alias.jump1
+      - ch.alias.jump2
+      - ch.alias.jump3
+      - essentials.heal
+      - ch.alias.gmc
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: matter manipulation

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2777,6 +2777,29 @@ groups:
...
+      - ch.alias.gms
+      - essentials.fly
+      - ch.alias.invis
+      - ch.alias.vis
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: invisibility

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -2777,6 +2777,29 @@ groups:
...
+      - worldedit.navigation.thru.command
+      - essentials.teleport
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: empty line

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -2777,6 +2777,29 @@ groups:
...
+      - essentials.heal
+      - ch.alias.gmc
+      - ch.alias.gms
+      - essentials.fly
+      - ch.alias.invis
+      - ch.alias.vis
+      - worldedit.navigation.thru.command
+      - essentials.teleport
+      
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: empty line

File: PermissionsEx/permissions.yml [hunk 5]
```diff
@@ -2777,6 +2777,29 @@ groups:
...
+      - ch.alias.jump2
+      - ch.alias.jump3
+      - essentials.heal
+      - ch.alias.gmc
+      - ch.alias.gms
+      - essentials.fly
+      - ch.alias.invis
+      - ch.alias.vis
+      - worldedit.navigation.thru.command
+      - essentials.teleport
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: teleportation

File: PermissionsEx/permissions.yml [hunk 6]
```diff
@@ -2777,6 +2777,25 @@ groups:
...
+      - ch.alias.jump0
+      - ch.alias.jump1
+      - ch.alias.jump2
+      - ch.alias.jump3
+      - ch.alias.gmc
+      - ch.alias.gms
+      - essentials.fly
+      - essentials.teleport
+      
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: delete blank line

File: PermissionsEx/permissions.yml [hunk 7]
```diff
@@ -2777,6 +2777,25 @@ groups:
...
+      - ch.alias.run4
+      - ch.alias.jump0
+      - ch.alias.jump1
+      - ch.alias.jump2
+      - ch.alias.jump3
+      - ch.alias.gmc
+      - ch.alias.gms
+      - essentials.fly
+      - essentials.teleport
+
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: delete blank line

**Pull request merged**